comarev/comarev-dashboard
# Issue 48 |  Fix broken tests

Closes: https://github.com/comarev/comarev-dashboard/issues/48

#### What?

- Fixes old tests and sets a new environment to e2e tests.

#### Why?

- To fix [issue](https://github.com/comarev/comarev-dashboard/issues/48), improving test coverage and code quality.

#### How to test it?

- You can run `yarn test` and see if everything is passing now and also `yarn test:e2e` ([check the steps to proper run cypress](https://github.com/comarev/comarev-dashboard/pull/66/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R73)) to check the new e2e test.

![image](https://user-images.githubusercontent.com/47646621/157716516-4eea57ba-d264-4a18-8ec3-d967304693a5.png)
![image](https://user-images.githubusercontent.com/47646621/157716926-bad50585-b1cd-4c97-829e-a0b050deab10.png)
